package.json

```diff
@@ -31,7 +33,12 @@
     "cypress:open": "cypress open",
     "cypress:run": "cypress run -c video=false",
     "cypress:ci": "start-server-and-test start:ci http-get://localhost:8080 cypress:run",
-    "cypress:prepare-ci": "bin/cypress-prepare-ci"
+    "cypress:prepare-ci": "bin/cypress-prepare-ci",
+    "s3": "EXTRA_CORS_ALLOWED_ORIGINS=http://localhost:5000 MOTO_ALLOW_NONEXISTENT_REGION=True moto_server -H 0.0.0.0",
+    "s3:install": "brew install moto",
+    "s3:reset": "curl -X POST http://localhost:5000/moto-api/reset",
+    "sqs": "java -jar elasticmq-server-1.3.9.jar",
```

Comment:
I'd rather have sqs offline set up the same way as other projects our team manages, with the docker compose file.

I _hate_ the idea of running extra processes in order to run EDSC locally. [This PR for serverless offline sqs](https://github.com/CoorpAcademy/serverless-plugins/pull/117) shows that it is possible to disable sqs offline. So we'd get the benefit of queues locally, but only if the dev wants them. I don't think that is happening, but it is easy enough to open a new PR that is up to date and install the forked version of the plugin